doc/manual/rule_uid_schema.md

```diff
@@ -0,0 +1,189 @@
+   case string after the last dot of the full name)
+ * Optional **Version**: a counter value representing the current version of the
+   rule content. If content of the rule changes without changing the underlying
+   logic for validation, this counter increases. If the change in the rule
+   forces a modification in the validation logic, the change shall be treated as
+   a new rule.
+
+To provide a visual description for a rule UID:
+
+```
+# Without version
+<emanating-entity>:<standard>:x.y.z:rule_set.for_rules.rule_name
+# With Version
+<emanating-entity>:<standard>:x.y.z:rule_set.for_rules.rule_name:2
+```
+
+> Third party rule UID creators (i.e., emanating entities different than ASAM)
+> should still fill all the concepts above. If that is not possible, concepts
+> shall be left blank, but separation `:` is still required (i.e.,
+> `example.com:::rulename`is valid) 
+
+UID are designed to be queried, e.g., implementations may use UNIX pattern
+matching.
+
+### Full Pattern
+
+```
+^
+ (?P<ENTITY>(\w+(\.\w+)+)):
+ (?P<STANDARD>([a-z]+))?:
+ (?P<STD_VERSION>([0-9]+(\.[0-9]+)+))?:
+ (?P<RULESET>(([a-z][\w_]*)\.)*)(?P<NAME>[a-z][\w_]*)
+ (:(?P<RULE_VERSION>(\d+)))?
+$
+```
+
+### ASAM Concepts
+
+ * Emanating Entity: `asam.net`
+ * Standards:
+   * Quality Checker Framework: `asam.net:qc`
+   * OpenDRIVE: `asam.net:xodr`
+     * v1.4.0: `asam.net:xodr:1.4.0`
+     * v1.5.0: `asam.net:xodr:1.5.0` 
+     * v1.6.0: `asam.net:xodr:1.6.0`
+     * v1.6.1: `asam.net:xodr:1.6.1`
+     * v1.7.0: `asam.net:xodr:1.7.0`
+     * v1.8.0: `asam.net:xodr:1.8.0`
+   * OpenSCENARIO XML: `asam.net:xosc`
+     * v1.0.0: `asam.net:xosc:1.0.0` 
+     * v1.1.0: `asam.net:xosc:1.1.0`
+     * v1.1.1: `asam.net:xosc:1.1.1`
+     * v1.2.0: `asam.net:xosc:1.2.0` 
+   * OpenSCENARIO DSL: `asam.net:osc`
+     * v2.0.0: `asam.net:osc:2.0.0`
+
+Rule full names are defined in domain-specific Quality Checker subgroups. Third
+party emanating entities should use the standard and definition setting
+concepts defined by ASAM. 
+
+## Example
+
+Lets start with an example for a rule UID, and lets break down its concepts. The
+example references a rule listed in the subsection **Rules** of one of the ASAM
+Standards for OpenDRIVE. The Rules subsections were introduced for the first
+time in revision 1.6 of the standard. **Please notice that the full name of the
+rule may change in the future**.
+
+> [ASAM OpenDRIVE Standard 1.6.0, Chapter 7.2 - Road Reference
+> Line](https://releases.asam.net/OpenDRIVE/1.6.0/ASAM_OpenDRIVE_BS_V1-6-0.html#_road_reference_line),
+> subsection Rule, first point: 
+> > **Each road shall have a reference line**
+
+we can identify the following concepts:
+
+ * **emanating entity**: ASAM `asam.net`
+ * **standard**: OpenDRIVE `xodr`
+ * **definition settings**: the rule firstly appeared in `1.6.0`, the first
+   revision to introduce the subsections rules
+ * **Full name**:
+    * **rule-set**: Road Planar Geometry definitions `road.planview.geometry`
```

Comment:
I would not use **planview** here, as this layer has no rules in Opendrive and is only the "group tag" for the actual geometry elements.

What about rule-set for objects/signals? 
The complete xodr path would actually be  **road.objects.object** 
Do we want the complete path or a short form?

Reply:
I agree that I would not use planview, and in general the example is problematic: it is based **on a rule uid  that still has not been confirmed** by the xodr subgroup.

I think a **good approach is to open an issue on this document**, that states that all the examples should be changed and to use an existing and confirmed rule uid as soon as it available.

> Do we want the complete path or a short form?

I don't have a answer... Actually in the subgroup we already voted against using the xml structure for the rule-set, which is in line with avoiding _planview_ tags. The example still contains it because of aforementioned reasons and technical debt (the first time I wrote it I used the xml schema, before subgroup voted against it).

Reply:
One thing/one question regarding the shortened rule set path. What is exactly the schema in OpenDRIVE and/or other XML based structures? 

`road.objects.object` --> simply `object` (Remove everything what's not needed, because `object` can only be found with `road.objects` ? May be a little be nasty when talking about `outline`, because this can be in `road.objects.object.outline` or within `road.objects.object.outlines.outline`.

And some of the rules may apply to both variants. However, a shorter version would be nice ;-) 
Probably that is something, which need to be decided by the standard or the Checker Library itself?

Reply:
As for now I think we delegated the decision about rule-set path to standards. The apporach taken by @MircoNierenz (if I understand correctly) is to try to adhere to xml schema, but removing the "empty container" xpath elements. So `road.objects.object` may be `road.object`. But this still needs to be decided in the subgroup.